code/modules/mining/equipment_locker.dm

```diff
@@ -15,51 +17,116 @@
 		access_chemistry,
 		access_bar,
 		access_research,
-		access_ce,
-		access_virology
+		access_ce
```

Comment:
I noticed that, just going to leave it as this until I actually start messing with the map for testing purposes.